nginx.conf

```diff
@@ -58,29 +58,20 @@ http {
     # adding some custom rules to strip out the html extension
     index index.html;
 
-    if (!-f "${request_filename}index.html") {
```

Comment:
I'm not really sure what this stuff that was deleted does, so maybe @m-richo is best placed to review this...